cli/commands/build.js

```diff
@@ -1,12 +1,24 @@
 const fs = require("fs");
+
 const path = require("path");
 const { format } = require("prettier");
 const { generator } = require("../../core/generator");
 
+const flatten = arr => arr.reduce((acc, val) => acc.concat(val), []);
+const mkdirp = dir => dir.split('/').reduce((path, currentDir) => {
```

Comment:
I don't see how this function is different than the mkdir function I have.

mkdirp:
https://github.com/clio-lang/clio/blob/91b9751e4ecf1bbac021c323f5b823398f2e5c29/cli/commands/build.js#L8-L15

mkdir:
https://github.com/clio-lang/clio/blob/91b9751e4ecf1bbac021c323f5b823398f2e5c29/cli/commands/build.js#L26-L31

Reply:
Yup. It does the same thing. Remove mine and use yours.

Reply:
Both are wrong lol. Had to write a new one.